resolvers/simple.resolver.js

```diff
@@ -22,16 +22,19 @@ module.exports = async (job, log, data) => {
 
   allScripts.forEach((script) => {
     let scriptContents = log.substr(log.indexOf(script));
-    scriptContents = scriptContents.split('\n').slice(1).join('\n');
-    scriptContents = scriptContents.substr(0, scriptContents.indexOf('" exited with ')).trim();
-    scriptContents = scriptContents.split('\n').slice(0, -1).join('\n');
-    scriptContents = scriptContents.trim();
+    const exitCode = /exited\swith\s(\d)/g.exec(scriptContents);
```

Comment:
`\d` will only accept a single digit, while exit codes could contain more than one digit.

Reply:
Yes I was thinking about this as well, I wasn't sure about the exit codes being more than one digit. Will update this.